### ISSUE TITLE
Kwave doesn't load icons

### ISSUE THREAD

**Issue by MechDR** [void-linux/void-packages#47668]
### Is this a new report?

Yes

### System Info

Void 6.5.13_1 x86_64 GenuineIntel uptodate rrrmFFFFFFFF

### Package(s) Affected

kwave-23.04.0_3

### Does a report exist for this bug with the project's home (upstream) and/or another distro?

Wouldn't know to be honest...

### Expected behaviour

Load everything, including icons.

### Actual behaviour

Doesn't load icons in the GUI.

### Steps to reproduce

Install or build from template, see for yourslef.

**Comment by MechDR** [comment]
Screenshot
![Screenshot_2023-12-10_03-45-03](https://github.com/void-linux/void-packages/assets/45944962/3b0f84dc-7601-4617-86a2-50137d0c115f)

**Comment by MechDR** [comment]
Terminal output

```
This is kwave v23.04.0 (compiled with KDE Frameworks 5.112.0)

Benchmarking memcpy methods (smaller is better):
	libc : 40115284
	linux kernel : 18673600
	MMX  : 29117436
	MMXEXT : 46199096
	SSE : 43667292
	AVX : 43206224
using -> 'linux kernel'


            zero   2.3 written by Thomas Eschenbacher
          volume   2.3 written by Thomas Eschenbacher
     stringenter   2.3 written by Thomas Eschenbacher
        sonagram   2.3 written by Thomas Eschenbacher
     selectrange   2.3 written by Thomas Eschenbacher
      saveblocks   2.4 written by Thomas Eschenbacher
      samplerate   2.3 written by Thomas Eschenbacher
         reverse   2.3 written by Thomas Eschenbacher
          record   2.6 written by Thomas Eschenbacher
        playback   2.4 written by Thomas Eschenbacher
     pitch_shift   2.3 written by Thomas Eschenbacher
    notch_filter   2.3 written by Dave Flogeras
       normalize   2.3 written by Thomas Eschenbacher
           noise   2.3 written by Thomas Eschenbacher
       newsignal   2.3 written by Thomas Eschenbacher
         lowpass   2.3 written by Thomas Eschenbacher
       insert_at   2.3 written by Thomas Eschenbacher
            goto   2.3 written by Thomas Eschenbacher
        fileinfo   2.3 written by Thomas Eschenbacher
      export_k3b   2.3 written by Thomas Eschenbacher
           debug   2.3 written by Thomas Eschenbacher
       codec_wav   2.3 written by Thomas Eschenbacher
       codec_ogg   2.3 written by Thomas Eschenbacher
       codec_mp3   2.3 written by Thomas Eschenbacher
      codec_flac   2.3 written by Thomas Eschenbacher
 codec_audiofile   2.3 written by Thomas Eschenbacher
     codec_ascii   2.3 written by Thomas Eschenbacher
       band_pass   2.3 written by Dave Flogeras
     amplifyfree   2.3 written by Thomas Eschenbacher
           about   2.3 written by Ralf Waspe & Gilles Caulier
--- 
 found 30 plugins

MenuNode 'New...': icon 'document-new' not found !
MenuNode 'Open': icon 'document-open' not found !
MenuNode 'Open Recent': icon 'document-open-recent' not found !
MenuNode 'Clear List': icon 'edit-clear-list' not found !
MenuNode 'Record': icon 'media-record' not found !
MenuNode '&Save': icon 'document-save' not found !
MenuNode 'As...': icon 'document-save-as' not found !
MenuNode 'Blocks...': icon 'document-save-all' not found !
MenuNode 'Close': icon 'document-close' not found !
MenuNode 'New Window': icon 'window-new' not found !
MenuNode 'Revert': icon 'document-revert' not found !
MenuNode 'Quit': icon 'application-exit' not found !
MenuNode 'Undo': icon 'edit-undo' not found !
MenuNode 'Redo': icon 'edit-redo' not found !
MenuNode 'Cut': icon 'edit-cut' not found !
MenuNode 'Copy': icon 'edit-copy' not found !
MenuNode 'Paste': icon 'edit-paste' not found !
MenuNode 'Crop': icon 'transform-crop' not found !
MenuNode 'Delete': icon 'edit-delete' not found !
MenuNode 'Add': icon 'list-add' not found !
MenuNode 'Delete': icon 'list-remove' not found !
MenuNode 'Flush': icon 'edit-clear' not found !
MenuNode 'File Properties...': icon 'document-properties' not found !
MenuNode 'Begin': icon 'go-first' not found !
MenuNode 'End': icon 'go-last' not found !
MenuNode 'Previous Page': icon 'kwave_player_start' not found !
MenuNode 'Next Page': icon 'kwave_player_end' not found !
MenuNode 'Scroll Right': icon 'kwave_player_fwd' not found !
MenuNode 'Scroll Left': icon 'kwave_player_rew' not found !
MenuNode 'Zoom In': icon 'kwave_zoom_in' not found !
MenuNode 'Zoom Out': icon 'kwave_zoom_out' not found !
MenuNode 'Zoom to Selection': icon 'kwave_viewmag' not found !
MenuNode 'Zoom to Whole Signal': icon 'kwave_viewmagfit' not found !
MenuNode 'Zoom to 100%': icon 'kwave_zoom_original' not found !
MenuNode 'Previous': icon 'kwave_player_start' not found !
MenuNode 'Rewind': icon 'kwave_player_rew' not found !
MenuNode 'Start': icon 'kwave_player_play' not found !
MenuNode 'Loop': icon 'kwave_player_loop' not found !
MenuNode 'Pause': icon 'kwave_player_pause' not found !
MenuNode 'Continue': icon 'kwave_player_pause_2' not found !
MenuNode 'Stop': icon 'kwave_player_stop' not found !
MenuNode 'Forward': icon 'kwave_player_fwd' not found !
MenuNode 'Next': icon 'kwave_player_end' not found !
MenuNode 'Prepare...': icon 'configure' not found !
MenuNode 'Start Now!': icon 'media-record' not found !
MenuNode 'Add...': icon 'list-add' not found !
MenuNode 'Delete': icon 'list-remove' not found !
MenuNode 'Load...': icon 'document-open' not found !
MenuNode 'Save...': icon 'document-save' not found !
MenuNode 'Volume': icon 'player-volume' not found !
MenuNode 'Fade In': icon 'fade_in.png' not found !
MenuNode 'Fade Out': icon 'fade_out.png' not found !
MenuNode 'Amplify Free': icon 'amplify_free.png' not found !
MenuNode 'Reverse': icon 'object-flip-horizontal' not found !
MenuNode 'Silence': icon 'edit-clear' not found !
MenuNode 'Noise': icon 'noise.png' not found !
MenuNode 'Next Window': icon 'go-next-view' not found !
MenuNode 'Previous Window': icon 'go-previous-view' not found !
MenuNode 'Playback': icon 'speaker' not found !
MenuNode 'Format': icon 'configure' not found !
MenuNode 'Source': icon 'audio-input-microphone' not found !
MenuNode 'Reset Toolbar': icon 'configure-toolbars' not found !
MenuNode 'Contents': icon 'help-contents' not found !
MenuNode 'About Kwave': icon 'kwave' not found !
MenuNode 'About KDE': icon 'kde' not found !
MenuNode 'Export to K3b Project...': icon 'application-x-k3b' not found !
MenuNode 'Enter Command': icon 'editor' not found !
```

**Comment by paper42** [comment]
Do you have breeze icons installed?

**Comment by MechDR** [comment]
@paper42 no, but I'll try that.

OK, some icons appeared, but not all.

![Screenshot_2023-12-14_00-19-40](https://github.com/void-linux/void-packages/assets/45944962/0051f020-41d4-4d60-a081-557bdff455f9)

**Comment by Luciogi** [comment]
On kde desktop 
everything looks alright 
![image](https://github.com/void-linux/void-packages/assets/84625092/e96a94b4-12ca-499d-ae82-2bf4be6db623)

**Comment by MechDR** [comment]
@Luciogi I tried building from the template as well, no icons load on my install. Using xfce.

Could you list icon packages that you're using so I can install them one by one and pinpoint the problem?

**Comment by Luciogi** [comment]
https://github.com/void-linux/void-packages/assets/84625092/20f68f24-5a74-4104-aece-1d87699ebc7f

**Comment by MechDR** [comment]
@Luciogi I did try what you suggested in the video, no change, except for the size of the buttons.

![Screenshot_2023-12-20_18-36-40](https://github.com/void-linux/void-packages/assets/45944962/e6b32234-6993-49b7-a421-fbe12bb7f7aa)

![Screenshot_2023-12-20_18-37-26](https://github.com/void-linux/void-packages/assets/45944962/dccbdce9-6570-4ad4-8a8c-0192933ea25d)

![Screenshot_2023-12-20_18-37-50](https://github.com/void-linux/void-packages/assets/45944962/ff6fe500-ec8b-4043-904b-396d12128163)

![Screenshot_2023-12-20_18-38-11](https://github.com/void-linux/void-packages/assets/45944962/3f03826f-8961-43bd-86f4-d41f296f0e4a)

And I ran kwave from terminal and posted the output (a few comments above), but here it is again.

````
This is kwave v23.04.0 (compiled with KDE Frameworks 5.112.0)

Benchmarking memcpy methods (smaller is better):
	libc : 43800696
	linux kernel : 20899116
	MMX  : 35853036
	MMXEXT : 45211768
	SSE : 48029748
	AVX : 44520508
using -> 'linux kernel'


            zero   2.3 written by Thomas Eschenbacher
          volume   2.3 written by Thomas Eschenbacher
     stringenter   2.3 written by Thomas Eschenbacher
        sonagram   2.3 written by Thomas Eschenbacher
     selectrange   2.3 written by Thomas Eschenbacher
      saveblocks   2.4 written by Thomas Eschenbacher
      samplerate   2.3 written by Thomas Eschenbacher
         reverse   2.3 written by Thomas Eschenbacher
          record   2.6 written by Thomas Eschenbacher
        playback   2.4 written by Thomas Eschenbacher
     pitch_shift   2.3 written by Thomas Eschenbacher
    notch_filter   2.3 written by Dave Flogeras
       normalize   2.3 written by Thomas Eschenbacher
           noise   2.3 written by Thomas Eschenbacher
       newsignal   2.3 written by Thomas Eschenbacher
         lowpass   2.3 written by Thomas Eschenbacher
       insert_at   2.3 written by Thomas Eschenbacher
            goto   2.3 written by Thomas Eschenbacher
        fileinfo   2.3 written by Thomas Eschenbacher
      export_k3b   2.3 written by Thomas Eschenbacher
           debug   2.3 written by Thomas Eschenbacher
       codec_wav   2.3 written by Thomas Eschenbacher
       codec_ogg   2.3 written by Thomas Eschenbacher
       codec_mp3   2.3 written by Thomas Eschenbacher
      codec_flac   2.3 written by Thomas Eschenbacher
 codec_audiofile   2.3 written by Thomas Eschenbacher
     codec_ascii   2.3 written by Thomas Eschenbacher
       band_pass   2.3 written by Dave Flogeras
     amplifyfree   2.3 written by Thomas Eschenbacher
           about   2.3 written by Ralf Waspe & Gilles Caulier
--- 
 found 30 plugins

MenuNode 'Previous Page': icon 'kwave_player_start' not found !
MenuNode 'Next Page': icon 'kwave_player_end' not found !
MenuNode 'Scroll Right': icon 'kwave_player_fwd' not found !
MenuNode 'Scroll Left': icon 'kwave_player_rew' not found !
MenuNode 'Zoom In': icon 'kwave_zoom_in' not found !
MenuNode 'Zoom Out': icon 'kwave_zoom_out' not found !
MenuNode 'Zoom to Selection': icon 'kwave_viewmag' not found !
MenuNode 'Zoom to Whole Signal': icon 'kwave_viewmagfit' not found !
MenuNode 'Zoom to 100%': icon 'kwave_zoom_original' not found !
MenuNode 'Previous': icon 'kwave_player_start' not found !
MenuNode 'Rewind': icon 'kwave_player_rew' not found !
MenuNode 'Start': icon 'kwave_player_play' not found !
MenuNode 'Loop': icon 'kwave_player_loop' not found !
MenuNode 'Pause': icon 'kwave_player_pause' not found !
MenuNode 'Continue': icon 'kwave_player_pause_2' not found !
MenuNode 'Stop': icon 'kwave_player_stop' not found !
MenuNode 'Forward': icon 'kwave_player_fwd' not found !
MenuNode 'Next': icon 'kwave_player_end' not found !
MenuNode 'Fade In': icon 'fade_in.png' not found !
MenuNode 'Fade Out': icon 'fade_out.png' not found !
MenuNode 'Amplify Free': icon 'amplify_free.png' not found !
MenuNode 'Noise': icon 'noise.png' not found !
````

BTW, apparently Breeze icons are a must in the package (fixed the New, Open, Save, Cut, etc. missing icons).

**Comment by Luciogi** [comment]
run this 
```
XDG_DATA_DIRS=/usr/local/share:/usr/share:~/.local/share kwave
```

**Comment by MechDR** [comment]
@Luciogi Nope, still no icons.
````
This is kwave v23.04.0 (compiled with KDE Frameworks 5.111.0)

Benchmarking memcpy methods (smaller is better):
	libc : 41928200
	linux kernel : 27836404
	MMX  : 39672648
	MMXEXT : 45717396
	SSE : 46791872
	AVX : 51466132
using -> 'linux kernel'


            zero   2.3 written by Thomas Eschenbacher
          volume   2.3 written by Thomas Eschenbacher
     stringenter   2.3 written by Thomas Eschenbacher
        sonagram   2.3 written by Thomas Eschenbacher
     selectrange   2.3 written by Thomas Eschenbacher
      saveblocks   2.4 written by Thomas Eschenbacher
      samplerate   2.3 written by Thomas Eschenbacher
         reverse   2.3 written by Thomas Eschenbacher
          record   2.6 written by Thomas Eschenbacher
        playback   2.4 written by Thomas Eschenbacher
     pitch_shift   2.3 written by Thomas Eschenbacher
    notch_filter   2.3 written by Dave Flogeras
       normalize   2.3 written by Thomas Eschenbacher
           noise   2.3 written by Thomas Eschenbacher
       newsignal   2.3 written by Thomas Eschenbacher
         lowpass   2.3 written by Thomas Eschenbacher
       insert_at   2.3 written by Thomas Eschenbacher
            goto   2.3 written by Thomas Eschenbacher
        fileinfo   2.3 written by Thomas Eschenbacher
      export_k3b   2.3 written by Thomas Eschenbacher
           debug   2.3 written by Thomas Eschenbacher
       codec_wav   2.3 written by Thomas Eschenbacher
       codec_ogg   2.3 written by Thomas Eschenbacher
       codec_mp3   2.3 written by Thomas Eschenbacher
      codec_flac   2.3 written by Thomas Eschenbacher
 codec_audiofile   2.3 written by Thomas Eschenbacher
     codec_ascii   2.3 written by Thomas Eschenbacher
       band_pass   2.3 written by Dave Flogeras
     amplifyfree   2.3 written by Thomas Eschenbacher
           about   2.3 written by Ralf Waspe & Gilles Caulier
--- 
 found 30 plugins

MenuNode 'Previous Page': icon 'kwave_player_start' not found !
MenuNode 'Next Page': icon 'kwave_player_end' not found !
MenuNode 'Scroll Right': icon 'kwave_player_fwd' not found !
MenuNode 'Scroll Left': icon 'kwave_player_rew' not found !
MenuNode 'Zoom In': icon 'kwave_zoom_in' not found !
MenuNode 'Zoom Out': icon 'kwave_zoom_out' not found !
MenuNode 'Zoom to Selection': icon 'kwave_viewmag' not found !
MenuNode 'Zoom to Whole Signal': icon 'kwave_viewmagfit' not found !
MenuNode 'Zoom to 100%': icon 'kwave_zoom_original' not found !
MenuNode 'Previous': icon 'kwave_player_start' not found !
MenuNode 'Rewind': icon 'kwave_player_rew' not found !
MenuNode 'Start': icon 'kwave_player_play' not found !
MenuNode 'Loop': icon 'kwave_player_loop' not found !
MenuNode 'Pause': icon 'kwave_player_pause' not found !
MenuNode 'Continue': icon 'kwave_player_pause_2' not found !
MenuNode 'Stop': icon 'kwave_player_stop' not found !
MenuNode 'Forward': icon 'kwave_player_fwd' not found !
MenuNode 'Next': icon 'kwave_player_end' not found !
MenuNode 'Fade In': icon 'fade_in.png' not found !
MenuNode 'Fade Out': icon 'fade_out.png' not found !
MenuNode 'Amplify Free': icon 'amplify_free.png' not found !
MenuNode 'Noise': icon 'noise.png' not found !
````

BTW, this is no isolated case, I tried installing Kwave on 2 other installs, one x86, the other x64, all of them running xfce, none of them show the lower toolbar icons. If breeze icon pack is installed, the upper toolbar icons are visible.

**Comment by Luciogi** [comment]
is void linux installed with musl or glib?

**Comment by MechDR** [comment]
@Luciogi glibc

**Comment by Luciogi** [comment]
Ok issue is reproducible in clean chroot

**Comment by MechDR** [comment]
@Luciogi OK :+1: .

**Comment by Luciogi** [comment]
Just an update: Sorry I am not getting time to find out what is causing this problem.

**Comment by MechDR** [comment]
@Luciogi That's OK, I've been busy myself. And I don't actually use this package, I was just browsing for some simple wave editor and this came up on my search. I eventually settled for ocenaudio.

**Comment by Luciogi** [comment]
Issue: QT framework uses `.svg` files, and source code is providing `.svgz` files, simple solution is to convert them to plain `.svg` files while packaging

**Comment by MechDR** [comment]
@Luciogi great, now they work 😊.

But, the app icon is still not working...

**Comment by classabbyamp** [comment]
the app icon is also an svgz i think

**Comment by MechDR** [comment]
Maybe that is in a different directory than the rest of them...

**Comment by classabbyamp** [comment]
fixed that one in b0b4a84a7ef44b73c627935f8580463d24b561b9

**Comment by MechDR** [comment]
Great 👍 😊.
